packages/core-ethereum/src/index.ts

```diff
@@ -283,41 +303,57 @@ export default class HoprCoreEthereum extends EventEmitter {
     if (!channel.destination.eq(this.getPublicKey())) {
       // delete operation before returning
       delete this.ticketRedemtionInChannelOperations[channelId]
-      throw new Error('Cannot redeem ticket in channel that is not to us')
+      return Promise.reject(new Error('Cannot redeem ticket in channel that is not to us'))
```

Comment:
Please document the replacement. Is this approach to be used everywhere in the future?